inscriptions/ISic000839.xml

```diff
@@ -187,18 +190,30 @@
             <graphic n="print" url="ISic000839.jpg" height="3680px" width="5520px">
                 <desc>I.Sicily with the permission of the Assessorato Regionale dei Beni Culturali e dell’Identità Siciliana - Dipartimento dei Beni Culturali e dell’Identità Siciliana</desc>
             </graphic>
-         </surface>
+        </surface>
     </facsimile>
     <text>
         <body>
             <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>Ἀτηδία <orig>ΠΟ<unclear>Λ</unclear>Π</orig><gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="2"/>ἄ<supplied reason="omitted">μ</supplied>εν<supplied reason="omitted">π</supplied>το<supplied reason="omitted">ς</supplied> <orig>ΟΥ</orig> <gap reason="lost" extent="unknown" unit="character"/> 
+                    <lb n="1"/><w n="5">Ἀτηδία</w> <orig n="10">ΠΟ<unclear>Λ</unclear>Π</orig><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/><w n="15">ἄ<supplied reason="omitted">μ</supplied>εν <supplied reason="omitted">π</supplied>το <supplied reason="omitted">ς</supplied></w> <orig n="20">ΟΥ</orig><gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
it's captured the word correctly (impressive given the mess here), but seems to have added a space in the middle, after εν and before the second supplied element.